src/sentry/relay/config/metric_extraction.py

```diff
@@ -115,20 +112,14 @@ def on_demand_metrics_feature_flags(organization: Organization) -> Set[str]:
 
 
 @metrics.wraps("on_demand_metrics._get_alert_metric_specs")
-def _get_alert_metric_specs(
-    project: Project, enabled_features: Set[str], prefilling: bool
-) -> List[HashedMetricSpec]:
-    if not ("organizations:on-demand-metrics-extraction" in enabled_features or prefilling):
+def _get_alert_metric_specs(project: Project, enabled_features: Set[str]) -> list[HashedMetricSpec]:
+    if not ("organizations:on-demand-metrics-extraction" in enabled_features):
         return []
 
-    metrics.incr(
-        "on_demand_metrics.get_alerts",
-        tags={"prefilling": prefilling},
-    )
+    metrics.incr("on_demand_metrics.get_alerts")
 
     datasets = [Dataset.PerformanceMetrics.value]
-    if prefilling:
-        datasets.append(Dataset.Transactions.value)
+    datasets.append(Dataset.Transactions.value)
```

Comment:
I assume `prefilling` is always `True`.

Reply:
wrt my other comment, it should be always false